pcp/versions/1.0/association.md

```diff
@@ -1,37 +1,32 @@
 Session Association
 ===
 
-PCP clients must associate their session with a broker before they can use
-its capabilities. Session Association is the process of establishing a connection
-between a client and broker and the broker associating a client URI with that connection.
+PCP clients must associate their underlying wire layer connection with a broker
+before they can use PCP capabilities. Session Association is the process of
+establishing a connection between a client and broker and the broker associating
+a client URI with that connection.
 
 This is done by completing a successful _associate_ transaction.
 
 Association Message Flow
 ---
 
 A client that wants to establish a session with a broker must 1) establish the
-underlying wire layer connection (e.g. WebSocket) and 2) send a *associate request*
-message to that PCP broker.
+underlying wire layer connection (e.g. WebSocket) and 2) send an *Associate Request*
+message to that PCP broker. The broker will then 3) reply with an
+*Associate Response*, indicating the outcome of the Session Association.
 
-In case the request is processed successfully, the broker will reply with a
-*associate response* message. The broker may ignore the request or reply with an
-*error message* in case of node identity mismatch (see
-[Broker Operation](#broker-operation)).
-
-Should a session already be associated with the given client URI, the broker
-should disconnect the older connection, and consider the newly-associated
-session to be authoritative for that URI.
+We describe how the broker must handle the Session Association in the
+[Broker Operation](#broker-operation)) section below.
 
 Associate Messages
 ---
 
-*Associate request* and *response* messages must have the envelope *message_type*
+*Associate Request* and *Response* messages must have the envelope *message_type*
 entry respectively equal to `http://puppetlabs.com/associate_request` and
-`http://puppetlabs.com/associate_response`. An Associate request message does not
```

Comment:
It's still accurate to say they don't require a Data Chunk. We don't reject messages that have a data chunk, but we'll ignore that section. But I suppose we can update the specification to say don't put one there, in case we want to later restrict it.